test/conformance/tests/sriov_operator.go

```diff
@@ -1514,3 +1508,26 @@ func waitForSRIOVStable() {
 		return isClusterReady
 	}, waitingTime, 1*time.Second).Should(BeTrue())
 }
+
+func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
+	cfg := sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{
+		Name:      "default",
+		Namespace: operatorNamespace,
+	}, &cfg)
+
+	Expect(err).ToNot(HaveOccurred())
+	if flagName == operatorNetworkInjectorFlag && *cfg.Spec.EnableInjector != flagValue {
```

Comment:
here you need to change that both the spec is not nill and the enable Injector is not nil

Reply:
We agreed to leave it like that